txpool/txpool_test.go

```diff
@@ -470,7 +474,46 @@ func TestTxPool_ErrorCodes(t *testing.T) {
 		})
 	}
 }
+func TestTx_MaxSize(t *testing.T) {
+	pool, err := NewTxPool(hclog.NewNullLogger(), false, forks.At(0), &mockStore{}, nil, nil)
+	pool.EnableDev()
+	assert.NoError(t, err)
+
+	tests := []struct {
+		name    string
+		address types.Address
+		succeed bool
+		size    uint64
+	}{
 
+		{
+			name:    "Tx_Data is greater than  MAX_SIZE",
+			address: types.Address{0x1},
+			succeed: false,
+			size:    132096,
+		},
+		{
+			name:    "Tx_Data is less than  MAX_SIZE",
```

Comment:
Just curious questions, double spaces between `than` and `MAX_SIZE` is intentional?

Reply:
No its not 😅

Reply:
Fixed it @Kourin1996